test/tracker.jl

```diff
@@ -9,6 +9,9 @@ using Tracker
             @test_throws ArgumentError AD.second_lowest(backend)
             @test_throws ArgumentError AD.hessian(backend, sum, randn(3))
         end
+        @testset "primal_value for array of tracked reals" begin
+            @test AD.primal_value([Tracker.gradient(sum, [1.0])[1][1]]) isa Vector{Float64}
```

Comment:
Wouldn't it be cleaner to construct `TrackedReal` directly? Tracker is really inconsistent as to whether or not it returns tracked types:

```suggestion
            @test AD.primal_value([Tracker.TrackedReal(1.0)]) isa Vector{Float64}
            @test AD.primal_value(fill(Tracker.TrackedReal(1.0), 1, 1)) isa Matrix{Float64}
```